src/MessageHandler/ActivityPub/Inbox/ActivityHandler.php

```diff
@@ -101,6 +101,16 @@ private function handle(?array $payload)
             }
             if (\is_array($payload['object'])) {
                 $payload = $payload['object'];
+                $actor = $payload['actor'] ?? $payload['attributedTo'] ?? null;
+                if ($actor) {
+                    $user = $this->manager->findActorOrCreate($actor);
```

Comment:
It's interesting that it's almost equal to line 96. Can a magazine also be part of attributedTo?

Reply:
ow nevermind, the actor is within the object payload.